docs/how-to-guides/auto-update-flatcar.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Auto-updating Flatcar Container Linux
+weight: 10
+---
+
+## Introduction
+
+At the moment, Lokomotive supports only [Flatcar Container Linux](https://www.flatcar-linux.org/) as
+the underlying operating system for nodes. While Flatcar can keep itself up to date, when running
+Kubernetes on top of it, we recommend using the [Flatcar Linux Update Operator
+(FLUO)](https://github.com/kinvolk/flatcar-linux-update-operator) to perform updates to avoid
+rebooting too many nodes at a time which could cause a service outage.
+
+FLUO is a supported Lokomotive component.
+
+This guide will show how to install it and disable its
+auto-update feature for specific nodes, which might be useful when you run services, which require
+special care before shutting down (e.g. storage clusters).
+
```

Comment:
Suggestion: I still think this should be mentioned for completeness.
```suggestion

NOTE: If you want to use FLUO component on non-Lokomotive cluster, make sure your Flatcar nodes has `locksmithd.service` systemd unit disabled to avoid nodes rebooting on their own. On Lokomotive this is done out of the box.
```